go_pprof_scraper/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2b13f6b1-d3ba-4254-93c0-2ceaf783cd85",
+  "app_id": "go-pprof-scraper",
```

Comment:
```suggestion
  "app_id": "go_pprof_scraper",
```

Reply:
It looks like the CI check doesn't like the new name: https://github.com/DataDog/integrations-extras/actions/runs/3393978080/jobs/5641957063#step:13:18, should I change it back or change it to something else?

Reply:
oops, forgot about that sorry, yes switch it back to `go-pprof-scraper` please